server/schedule/operator/create_operator.go

```diff
@@ -258,78 +85,36 @@ func CreateSplitRegionOperator(desc string, region *core.RegionInfo, kind OpKind
 	return NewOperator(desc, brief, region.GetID(), region.GetRegionEpoch(), kind, step)
 }
 
-func getRegionFollowerIDs(region *core.RegionInfo) []uint64 {
-	var ids []uint64
-	for id := range region.GetFollowers() {
-		ids = append(ids, id)
-	}
-	return ids
-}
-
-// removePeerSteps returns the steps to safely remove a peer. It prevents removing leader by transfer its leadership first.
-func removePeerSteps(cluster Cluster, region *core.RegionInfo, storeID uint64, followerIDs []uint64) (kind OpKind, steps []OpStep, err error) {
-	kind, steps, err = transferLeaderStep(cluster, region, storeID, followerIDs)
-	if err != nil {
-		return
-	}
-
-	steps = append(steps, RemovePeer{FromStore: storeID})
-	kind |= OpRegion
-	return
-}
-
-func transferLeaderStep(cluster Cluster, region *core.RegionInfo, storeID uint64, followerIDs []uint64) (kind OpKind, steps []OpStep, err error) {
-	if region.GetLeader() != nil && region.GetLeader().GetStoreId() == storeID {
-		kind, steps, err = transferLeaderToSuitableSteps(cluster, storeID, followerIDs)
-		if err != nil {
-			log.Debug("failed to create transfer leader step", zap.Uint64("region-id", region.GetID()), zap.Error(err))
-			return
-		}
-	}
-	return
-}
-
-// findNoLabelProperty finds the first store without given label property.
-func findNoLabelProperty(cluster Cluster, prop string, storeIDs []uint64) (int, uint64) {
-	for i, id := range storeIDs {
-		store := cluster.GetStore(id)
-		if store != nil {
-			if !cluster.CheckLabelProperty(prop, store.GetLabels()) {
-				return i, id
+// CreateMergeRegionOperator creates an operator that merge two region into one.
+func CreateMergeRegionOperator(desc string, cluster Cluster, source *core.RegionInfo, target *core.RegionInfo, kind OpKind) ([]*Operator, error) {
```

Comment:
Is it possible to create the merge operators just like the others by using `build`?

Reply:
Seems kind of beyond the capabilities of the builder :(